compose.example.yml

```diff
@@ -0,0 +1,21 @@
+---
+services:
+  sml2mqtt:
+    image: spacemanspiff2007/sml2mqtt
+    container_name: sml2mqtt
+    init: true
+    restart: unless-stopped
+    mem_limit: 64m
+    volumes:
+      # Subdirectory ./data contains config.yml and log files
+      - "./data:/sml2mqtt"
+    devices:
+      # Replace with the location to your device
+      - /dev/serial/by-id/<ID-OF-SERIAL-DEVICE-TO-METER>
```

Comment:
Can we have both (`dev/serial/tty...` and `by-id`) and add a comment that `by-id` is recommended?

Reply:
Done